app/javascript/components/StudioAssessmentQuestionView/index.js

```diff
@@ -0,0 +1,213 @@
+import React from 'react';
```

Comment:
I don't think this is supposed to be here. Must have been from an earlier version of master

Reply:
Oh wait I think there's a React component at the bottom of the file

Reply:
Do you mean the entire file?

Reply:
yes